README.md

```diff
@@ -96,6 +96,7 @@ Session store-related settings:
 | - | - | - |
 | `SESSION_STORE_PATH` | "/var/lib/authservice/data.db" | Path to local session store. Backed by BoltDB. |
 | `SESSION_MAX_AGE` | "86400" | Time in seconds after which sessions expire. Defaults to a day (24h). |
+| `SESSION_SAME_SITE` | "Lax" | Allow you to specify the SameSite attribute of the Set-Cookie HTTP response header. Check details of SameSite attribute [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite). Its value could be "None", "Lax" or "Strict"  |
```

Comment:
Can you use the following text for description?

SameSite attribute of the session cookie. Check details of SameSite attribute [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite). Its value can be "None", "Lax" or "Strict".

Reply:
done